packages/core/src/preview.ts

```diff
@@ -73,6 +80,8 @@ export const getPreviewString = (
           )
 
           executingArray.push(green(`${i + 1}. ${actionStr}`))
+        } else if (isSystemDeploymentElement(element)) {
+          executingArray.push(green(`${i + 1}. Sphinx & Gnosis Safe Contracts`))
```

Comment:
Honestly, we should probably be more aggressive with this since the user may have to pay quite a bit to deploy all the Safe contracts, and since our system may not be compatible with the network at all. I'm fine with just opening a ticket for this, though, if you don't want to address it now.

Reply:
Yeah you're right. Added a personal note to create a Linear ticket for this